packages/web3-eth-iban/src/index.ts

```diff
@@ -15,218 +15,9 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { toChecksumAddress, isAddress, leftPad, hexToNumber, HexString } from 'web3-utils';
-import { InvalidAddressError } from 'web3-errors';
-import { IbanOptions } from './types';
+import { Web3Iban } from './web3_iban';
 
-export class Iban {
```

Comment:
Why we need this refactor to new file?

Reply:
To have a consistent pattern around the code base. To have classes in files as their names. It's easy to search in the code.